Code/GraphMol/Fingerprints/MHFP.cpp

```diff
@@ -92,21 +92,26 @@ std::vector<uint32_t> MHFPEncoder::FromArray(const std::vector<uint32_t>& vec) {
 std::vector<std::string> MHFPEncoder::CreateShingling(
     const ROMol& mol, unsigned char radius, bool rings, bool isomeric,
     bool kekulize, unsigned char min_radius) {
+  RWMol tmol(mol);
+  if (kekulize) {
```

Comment:
What if this fails?

Reply:
Perhaps a general comment that kekulize will throw an exception if the input is not kekulizable.